src/bp/core/services/cms.ts

```diff
@@ -597,24 +597,26 @@ export class CMSService implements IDisposeOnExit {
     }, {})
   }
 
-  async renderElement(contentId, args, eventDestination: IO.EventDestination) {
+  async renderElement(contentId: string, args, eventDestination: IO.EventDestination) {
     const { botId, channel } = eventDestination
     contentId = contentId.replace(/^#?/i, '')
     let contentTypeRenderer: ContentType
 
+    const translateFormData = async (formData: object): Promise<object> => {
+      const defaultLang = (await this.configProvider.getBotConfig(eventDestination.botId)).defaultLanguage
```

Comment:
Just to be sure, is botconfig cached ?
+ why do you get this here and pass this to `getOriginalProps` when that function as access to `this` and therefore to defaultLanguage

Reply:
It is cached as with all other files from the ghost. I used the exact same implementation as before, just moved it there because I had to use it in 2 places. 

I left the method there because you need the botId and the user's language, which are extracted from the event, which is a parameter in the parent function